document-service-api/src/lib/config.js

```diff
@@ -19,6 +21,23 @@ module.exports = {
       },
     },
   },
+  docs: {
+    api: {
+      path: process.env.DOCS_API_PATH || path.join(__dirname, '..', 'api'),
+    },
+  },
+  fileUpload: {
+    maxSizeInBytes: Number(process.env.FILE_MAX_SIZE_IN_BYTES || 1000),
+    mimeTypes: [
+      'application/pdf', // pdf
+      'application/msword', // doc
+      'application/vnd.openxmlformats-officedocument.wordprocessingml.document', // docx
+      'image/tiff', // tiff
```

Comment:
could be image/tif as well

Reply:
As with jpg, seemingly not https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types